pkg/controller/servicemesh/controlplane/htpasswd.go

```diff
@@ -32,9 +32,9 @@ func (r *controlPlaneInstanceReconciler) patchHtpasswdSecret(ctx context.Context
 			log.Error(err, "failed to generate the HTPasswd password")
 			return err
 		}
-		h := sha1.New()
+		h := sha256.New()
 		h.Write([]byte(rawPassword))
-		auth = "internal:{SHA}" + base64.StdEncoding.EncodeToString(h.Sum(nil))
+		auth = "internal:{SHA-2}" + base64.StdEncoding.EncodeToString(h.Sum(nil))
```

Comment:
Is this correct?

When I run `htpasswd` locally, I see a different output (`-2` means sha256 and `-s` means sha-1):
```
[jwendell@jw-p1 h]$ htpasswd -n -2 -b internal random_password
internal:$5$RY8UyNHMWKk6b41z$kl2uJdF1Ns5xAeIRz8CrBbsMlESWhX3MgyxShG2fDD1

[jwendell@jw-p1 h]$ htpasswd -n -s -b internal random_password
internal:{SHA}1q7M/obgg9a8tcLtJFm4p+T0n+w=
```

Reply:
+1

`$5$` seems to be the correct prefix for SHA-256 passwords (see [here](https://github.com/tg123/go-htpasswd/blob/v1.2.0/cryptsha.go#L20), for example)

Reply:
Thank you for pointing that out, I wasn't sure where to find the standard prefix. I will amend that in a new commit. Testing grafana now.